Mathlib/CategoryTheory/Limits/FunctorCategory.lean

```diff
@@ -445,3 +446,127 @@ def colimitIsoSwapCompColim [HasColimitsOfShape J C] (G : J ⥤ K ⥤ C) :
 #align category_theory.limits.colimit_iso_swap_comp_colim CategoryTheory.Limits.colimitIsoSwapCompColim
 
 end
+
+-- variable {C : Type u} [Category.{v} C] {D : Type u'} [Category.{v'} D]
+
+--def typefunc (C : Type u) [Category.{v} C] : Type (max v (u + 1)) := C ⥤ (Type u)
+
+--instance blah : Quiver (typefunc C) := sorry
+
+namespace FunctorProd
+
+/-- `prod X Y` is the explicit binary product of simplicial sets `X` and `Y`. -/
+def prod (F G : C ⥤ (Type u)) : C ⥤ (Type u) where
```

Comment:
Instead of `CategoryTheory.FunctorProd.prod`, I would rather see that as `CategoryTheory.Types.functorProd`.

Reply:
Sorry for the late reply (I had midterms). I'm coming back to this PR, and I'm wondering what you mean by this. Do you think I should change the namespace to `Types` instead of `FunctorProd`, or are you saying I should move this code to `Mathlib/CategoryTheory/Types.lean`?

Reply:
`Types.functorProd` and `FunctorToTypes.prod` would be ok. (Presumably `FunctorToTypes.prod` is better because we already have the `FunctorToTypes` namespace; I am sorry for not thinking about this better name in the first place). Then, as I am suggesting below, as this deals with specific shapes of (co)limits for functors to the category of types, the new code should be moved to a new `Limits.Shapes.FunctorToTypes`, which should also fix the linter issue.

Reply:
`FunctorToTypes.prod` seems like the best option, thanks.